File: swagger/models/add_client_to_class_response.py
from .base_model import BaseModel

from .add_client_to_class_visit import AddClientToClassVisit

class AddClientToClassResponse(BaseModel):
	"""
	no description available

	Attributes:

		visit
		Contains information about the created visit.
	"""

	input_map = {
		'Visit': ('visit', AddClientToClassVisit),
		}

	output_map = {
		'visit': 'Visit',
		}

	visit = None

File: swagger/Models/AddClientToClassResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class AddClientToClassResponse extends BaseModel
{
	/**
	 * Contains information about the created visit.
	 * 
	 * @var AddClientToClassVisit 
	 */
	public $visit;

	protected function getInputMap()
	{
		return [
			'Visit' => ['visit', AddClientToClassVisit::class, null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'visit' => ['Visit', AddClientToClassVisit::class, null],
			];
	}
}
